hyperspy/_signals/signal1d.py

```diff
@@ -251,16 +251,14 @@ def _shift1D(data, **kwargs):
     size = kwargs.get('size', 2)
```

Comment:
`offset`, `scale` and `size` are not necessary anymore, remove them?

Reply:
indeed! They are now removed